### PR TITLE
Akihiro suda containerd test

### PR DESCRIPTION
**Pull request by rn**
This is an updated version of https://github.com/linuxkit/linuxkit/pull/1906 with the packages pushed and hashes updated

@AkihiroSuda I tried to push the update to your PR but got access denied (understandably)

### CODE REVIEW

File: test/cases/040_packages/003_containerd/test-containerd.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  - name: sysctl
+    image: "linuxkit/sysctl:225c52c2d6f04a040663bac84cabf81825027f64"
+  - name: test
+    image: "linuxkit/test-containerd:13debb1a62bfce592dd22250df953bbf6763fc50"
+  - name: poweroff
+    image: "linuxkit/poweroff:a8f1e4ad8d459f1fdaad9e4b007512cb3b504ae8"
+trust:
+  image:
+    - linuxkit/kernel
+    - linuxkit/binfmt
```
Comment (justincormack) [inline test/cases/040_packages/003_containerd/test-containerd.yml]: this should trust all of linuxkit org

File: test/cases/040_packages/003_containerd/test.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+# SUMMARY: Check that the sysctl config works
```
Comment (justincormack) [inline test/cases/040_packages/003_containerd/test.sh]: summary should be updated

File: test/cases/040_packages/003_containerd/test.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+. "${RT_PROJECT_ROOT}/_lib/lib.sh"
+
+clean_up() {
+	find . -iname "test-containerd*" -not -iname "*.yml" -exec rm -rf {} \;
+}
+trap clean_up EXIT
+
+# Test code goes here
+moby build test-containerd
+RESULT="$(linuxkit run qemu -mem 2048 test-containerd)"
```
Comment (justincormack) [inline test/cases/040_packages/003_containerd/test.sh]: don't specify `qemu`?

**Comment by rn** [comment]
forced pushed updates including fix to the test failure

**Comment by rn** [comment]
force pushed an update to the test YAML with latest package versions

File: src/cmd/linuxkit/run_hyperkit.go [hunk 4]
```diff
@@ -42,7 +43,7 @@ func runHyperKit(args []string) {
...
 	vsockports := flags.String("vsock-ports", "", "List of vsock ports to forward from the guest on startup (comma separated). A unix domain socket for each port will be created in the state directory")
-	networking := flags.String("networking", networkingDockerForMac, "Networking mode. Valid options are 'docker-for-mac', 'vpnkit[,socket-path]' and 'none'. 'docker-for-mac' connects to the network used by Docker for Mac. 'vpnkit' connects to the VPNKit socket specified. If socket-path is omitted a new VPNKit instance will be started and 'vpnkit_eth.sock' will be created in the state directory. 'none' disables networking.`")
+	networking := flags.String("networking", networkingDockerForMac, "Networking mode. Valid options are 'docker-for-mac', 'vpnkit[,socket-path]', 'vmnet' and 'none'. 'docker-for-mac' connects to the network used by Docker for Mac. 'vpnkit' connects to the VPNKit socket specified. If socket-path is omitted a new VPNKit instance will be started and 'vpnkit_eth.sock' will be created in the state directory. 'vmnet' uses the Apple vmnet framework, requires root/sudo. 'none' disables networking.`")
```
Comment (riyazdf) [inline src/cmd/linuxkit/run_hyperkit.go]: I think this needs a rebase

**Comment by rn** [comment]
OK, this is finally passing after more changes

**Pull request merged**